content/shmem_ctx_create.tex

```diff
@@ -1,5 +1,5 @@
 \apisummary{
-    Create a communication context locally.
+    Create a communication context.
 }
```

Comment:
Minor edit comment: should this be `Creates` with singular noun? For the shmem_sync api summary, we are using `Registers ... `. Just a question of whether we should unify these instances?

Reply:
The tense is consistent across all of the APIs in the contexts section, but not consistent across the whole spec.  Seems like it's best to leave this as-is for now and try to resolve holistically in a separate edit.